Update K8s quickstart to reflect Dashboard update (#484)
catalyst-cloud/catalystcloud-docs: source/kubernetes/quickstart.rst

Select the **Network** tab to configure the cluster's network access.

To enable access to the Kubernetes API from the public Internet, select
'Accessible with public floating IP' from the **Floating IP** drop-down list.

> For the quickstart, we will leave the **Allowed CIDRs** field empty.

Reviewer: ```suggestion
For this guide, we will leave the **Allowed CIDRs** field empty.
```

.. image:: _containers_assets/quickstart-network.png
    :align: center

That is everything you need to configure for this guide,
so press the **Submit** button to create the cluster.

You will be returned to the **Clusters** page, where you can monitor the state
of your Kubernetes clusters. Our new cluster should be listed,

> in ``CREATE_IN_PROGRESS`` state, refresh the browser to see the updated status.

Reviewer: ```suggestion
in ``CREATE_IN_PROGRESS`` state. If the cluster is not yet listed, refresh the page.
```

Author: Even if it is listed, the status won't update to `CREATE_COMPLETE` without refreshing. I'll have a think about the wording on this.

Reviewer: You're right. Guess we should also consider putting in an auto-refresh for this on the dashboard side, as well (if possible).

Author: Updated

As the Kubernetes dashboard of our cluster is not directly accessible from the Internet,
we will use ``kubectl`` to gain access to the Kubernetes dashboard.

First, ensure you have downloaded the **kubeconfig** file from the Catalyst Cloud
dashboard using the **Download Kubeconfig** button.

.. note::

>   Your browser may download the **kubeconfig** file to another location. Specific
>   browser download locations are out of scope for this document, you can also locate these
>   within your browser.

Reviewer: ```suggestion
  Your browser may download the **kubeconfig** file to another location.
  If you wish to download the **kubeconfig** file to a specific location
  (and not the default location for your operating system), replace ``.../Downloads``
  in the below file paths with your desired download location.
```

The kubeconfig file contains the required metadata used to authenticate
with the Kubernetes cluster.

.. tabs::

    .. group-tab:: Linux / macOS

      In a command line terminal environment (such as ``bash``) export the ``KUBECONFIG``
      environment variable, to configure ``kubectl`` to connect to your cluster.

      .. code-block:: bash

        export KUBECONFIG="${HOME}/Downloads/quickstart1_kubeconfig"

>       Alternatively, if you have no other Kubernetes clusters, you could move the downloaded file `quickstart1_kubeconfig` to the default location for ``kubectl``, in `$HOME/.kube/config`.

Reviewer: ```suggestion
      Alternatively, if you have no other Kubernetes clusters, you could move the downloaded file
      (``quickstart1_kubeconfig``) to the default location for ``kubectl``, in ``$HOME/.kube/config``.
```

      .. code-block:: bash

>         mv ${HOME}/Downloads/quickstart1_kubeconfig ${HOME}/.kube/config

Reviewer: Ensure `${HOME}/.kube` exists. Also, quote things to protect against spaces causing problems.

```suggestion
        mkdir -p "${HOME}/.kube/config" && mv "${HOME}/Downloads/quickstart1_kubeconfig" "${HOME}/.kube/config"
```

Reviewer: This suggestion was slightly incorrect. Just updated it.

    .. group-tab:: Windows (PowerShell)

>       In a PowerShell environment, define the ``KUBECONFIG`` environment variable, to configure ``kubectl``

Reviewer: ```suggestion
      In a PowerShell environment, define the ``KUBECONFIG`` environment variable to configure ``kubectl``
```

      to connect to your cluster.

      .. code-block:: powershell

>         $Env:KUBECONFIG = \Users\<username>\Downloads\quickstart1_kubeconfig

Reviewer: ```suggestion
        $Env:KUBECONFIG = $env:USERPROFILE\Downloads\quickstart1_kubeconfig
```

    .. group-tab:: Windows (Command Prompt)

      Define the ``KUBECONFIG`` environment variable, to configure ``kubectl``
      to connect to your cluster using the downloaded file.

      .. code-block:: bat

>         set KUBECONFIG=\Users\<username>\Downloads\quickstart1_kubeconfig

Reviewer: ```suggestion
        set KUBECONFIG=%homedrive%%homepath%\Downloads\quickstart1_kubeconfig
```
